### ISSUE TITLE
yo node-webkit "Potentially unhandled rejection" on Mac

### ISSUE THREAD

**Issue by karan** [Dica-Developer/generator-node-webkit#62]
```
$ yo node-webkit

     _-----_
    |       |
    |--(o)--|   .--------------------------.
   `---------´  |    Welcome to Yeoman,    |
    ( _´U`_ )   |   ladies and gentlemen!  |
    /___A___\   '__________________________'
     |  ~  |
   __'.___.'__
 ´   `  |° ´ Y `

? What do you want to call your app? Allowed characters ^[a-zA-Z0-9]+$ oodo
? A little description for your app? 
? Would you mind telling me your username on GitHub? karan
? Do you want to install one of the node-webkit examples? No
     info Get GitHub informations
✔ Github informations successfully retrieved.
   invoke   node-webkit:download
? Do you want to download node-webkit? Yes
? Please specify which version of node-webkit you want download: (v0.10.5) 
     info     Check if version "v0.10.5" is available for download.
? Please specify which version of node-webkit you want download: v0.10.5
? Which platform do you wanna support? MacOS 64
     info     Creating folder structure for node-webkit source.
✔ Created: "resources/node-webkit"
✔ Created: "resources/node-webkit/MacOS64"
✔ Created: "tmp"
     info     Downloading node-webkit for MacOS64
 conflict     Wrong content type for MacOS64
Potentially unhandled rejection [3] Wrong content type for MacOS64 (WARNING: non-Error used)
```

I'm on node v0.10.35 and Mac x64 (Mavericks).

**Comment by mschaaf** [comment]
I get this when downloading the macosx 32 version. I will have a look at this.

**Comment by mschaaf** [comment]
his seems to be fixed with issue #61 and can be tested with the version in the repository.
